src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannParser.cs

```diff
@@ -48,7 +50,20 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     throw new IndexerException(indexerResponse, "We are being redirected to the login page. Most likely your session expired or was killed. Try testing the indexer in the settings.");
                 }
 
-                throw new IndexerException(indexerResponse, $"Unexpected response status {indexerResponse.HttpResponse.StatusCode} code from API request");
+                // Catch common http exceptions before parsing
```

Comment:
Why not do this in HttpIndexerBase so it takes for all indexers, Line 397 is where we throw common errors before parsing.